config/helm/chart/default/templates/Common/kubernetes-monitoring/clusterrole-kubernetes-monitoring.yaml

```diff
@@ -79,4 +79,15 @@ rules:
       - /livez
     verbs:
       - get
+  {{- if eq (default false .Values.olm) true}}
```

Comment:
```suggestion
  {{- if .Values.olm }}
```

Reply:
guess we have that check everywhere, can be left as is

Reply:
We have that check everywhere, where we check for olm == true :)